server/openapi.yml

```diff
@@ -61,6 +69,8 @@ paths:
       summary: ユーザ情報取得API
       description: ユーザIDでユーザ情報を取得する
       operationId: getUserByUserId
+      security:
+        - cookieAuth: []
```

Comment:
[Q]
ここにparameters指定はいらない？

Reply:
必要なさそう
https://swagger.io/docs/specification/authentication/

> * In case of OAuth 2, the scopes used in security must be previously defined in securitySchemes.
> * In case of OpenID Connect Discovery, possible scopes are listed in the discovery endpoint specified by openIdConnectUrl.
> * Other schemes (Basic, Bearer, API keys and others) do not use scopes, so their security entries specify an empty array [] instead.

Reply:
勘違いしていた。
このコメントの意味を今理解した。
https://github.com/atsuya02/dutch_account/pull/17#discussion_r931091931